test/validation/org/apache/cassandra/bridges/ctoolbridge/CToolBridge.java

```diff
@@ -137,9 +140,9 @@ public String readClusterLogs(String testName)
         String combinedResult = "";
         String existingFolder = CASSANDRA_DIR + "/build/test/logs/validation/ctoolbridge/" + testName;
 
-        if(ArchiveClusterLogs.checkForFolder(existingFolder))
+        if (ArchiveClusterLogs.checkForFolder(existingFolder))
         {
-            for(int count = 1; count <= nodeCount; count++)
+            for (int count = 1; count <= nodeCount; count++)
```

Comment:
Is there a technical need to do this? Because if not, we really should use node0.log for node 0, and so on. It will be too confusing for someone who is looking at the logs, and wants to ssh into one of the nodes.